lib/urlwatch/handler.py

```diff
@@ -174,8 +174,9 @@ def _generate_diff(self):
 
 
 class Report(object):
-    def __init__(self, urlwatch_config):
-        self.config = urlwatch_config.config_storage.config
+    def __init__(self, urlwatch_obj):
+        self.config = urlwatch_obj.config_storage.config
+        self.urlwatch_config = urlwatch_obj.urlwatch_config
```

Comment:
Is this change necessary?

Reply:
Or in other words, could we just set the data directory as additional parameter, not expose the whole config object?

Reply:
Not sure why not to expose the whole config (other features might require access to command line args or other paths like cache), but I will change this.

> could we just set the data directory as additional parameter
Note that I only renamed existing parameter and did not introduce any new one (`urlwatch_config` -> `urlwatch_obj`). Previous name was misleading (https://github.com/thp/urlwatch/blob/master/lib/urlwatch/main.py#L54)